docs/recipes/handlebars.md

```diff
@@ -23,33 +23,72 @@ $ bower install --save handlebars
 ```
 
 * You need this so you can include the Handlebars runtime in your page – even compiled templates depend on this. (You won't need to include the entire Handlebars library though.)
+  ```
+  // bower.json
+  {
+    "name": "MyApp",
+    "dependencies": {
+      ...
+    },
+    "devDependencies": {
+      ...
+    },
+    "overrides": {
+      "handlebars": {
+        "main": ["handlebars.runtime.js"]
+      }
+    }
+  }
+  ```
 * It's a good idea to verify you've installed the same version of Handlebars as the one used by internally by gulp-handlebars, to guarantee compatibility between the runtime and your compiled templates. Look in `node_modules/gulp-handlebars/package.json` under `"dependencies"` and check the handlebars version – if necessary, you can ask bower to install that specific version, e.g. `bower install --save handlebars#^1.3.0`.
 
 ### 2. Create a `templates` task
 
 ```js
 gulp.task('templates', () => {
-  return gulp.src('app/templates/**/*.hbs')
+  return gulp.src('app/templates/*.hbs')
+    .pipe($.plumber())
     .pipe($.handlebars())
     .pipe($.defineModule('plain'))
     .pipe($.declare({
       namespace: 'MyApp.templates' // change this to whatever you want
     }))
+    .pipe($.concat('templates.js'))
     .pipe(gulp.dest('.tmp/templates'));
 });
 ```
+This compiles `.hbs` files into `templates.js` file in the `.tmp/templates` directory.
 
-This compiles `.hbs` files into `.js` files in the `.tmp` directory.
+### 2.1 Using Partials
+
+```js
+gulp.task('partials', function() {
+  return gulp.src('app/templates/partials/*.hbs')
+    .pipe($.plumber())
+    .pipe($.handlebars())
+    .pipe($.wrap('Handlebars.registerPartial(<%= processPartialName(file.relative) %>, Handlebars.template(<%= contents %>));', {}, {
+      imports: {
+        processPartialName: function(fileName) {
+          // Escape the output with JSON.stringify
+          return JSON.stringify(path.basename(fileName, '.js'));
+        }
+      }
+    }))
+    .pipe($.concat('partials.js'))
+    .pipe(gulp.dest('.tmp/templates'));
+});
+```
+This compiles the `.hbs` files inside the `templates/partials` directory into `partials.js` file in the `.tmp/templates` directory.
 
 ### 3. Add `templates` as a dependency of both `html` and `serve`
```

Comment:
This title needs to include the `partials` task (so "Add `templates` and `partials`…").